src/components/NoteDialog/NoteDialog.scss

```diff
@@ -0,0 +1,157 @@
+@import "constants/style";
+@import "components/Note/Note.scss";
+
+.note-dialog__header {
+  font-size: 48px;
+  color: $color-white;
+  width: fit-content;
+  border-bottom: solid 3px var(--accent-color);
+  padding-bottom: $padding--default;
+  margin: $margin--default 0 $margin--large 0;
+  letter-spacing: $letter-spacing--large;
+  line-height: $line-height--large;
+}
+
+.note-dialog__note {
+  padding: $padding--default $padding--medium;
+  position: relative;
+  border-radius: $note__border-radius;
+  background-color: $color-white;
+  box-shadow: 0 6px 9px 0 $note__box-shadow-color;
+  transition: transform 0.3s ease-in-out, box-shadow 0.3s ease-in-out;
+  list-style-type: none;
+  z-index: $note-dialog-z-index;
+  min-width: calc(240px + 10vw);
+  width: calc(100vw - 36px);
+  max-width: 380px;
+}
+@media (prefers-color-scheme: dark) {
+  .note-dialog__note {
+    background-color: $color-dark-mode-note;
+    box-shadow: 0 6px 9px 0 $note__box-shadow-color-dark-mode;
+  }
+}
+@media #{$menu-desktop} {
+  .note-dialog__note {
+    max-width: 440px;
+    width: calc(100vw - 32px);
+  }
+}
+
+.note-dialog__note--own-card::before {
+  content: "";
+  display: block;
+  position: absolute;
+  left: 0;
+  width: $note__indicator-width;
+  height: $note__indicator-height;
+  border-radius: 0 $note__indicator-width $note__indicator-width 0;
+  background: var(--accent-color);
+}
+
+.note-dialog__content {
+  display: flex;
+  width: 100%;
+}
+
+.note-dialog__text {
+  background: none;
+  margin: 0;
+  min-height: 3 * $line-height--medium;
+  width: 100%;
+  color: $color-black;
+  font-size: $text-size;
+  letter-spacing: $letter-spacing--medium;
+  line-height: $line-height--medium;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  margin-bottom: $note__text-margin-bottom;
+}
+.note-dialog__text:focus {
+  outline: 1px solid $color-dark-gray;
+  outline-offset: 6px;
+}
+@media (prefers-color-scheme: dark) {
+  .note-dialog__text {
+    color: $color-white;
+  }
+  .note-dialog__text:focus {
+    outline: 1px solid $color-white;
+  }
+}
+
+.note-dialog__author {
+  display: flex;
+  align-items: center;
+  // reset css from figure-element:
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
```

Comment:
You can delete `font-size: 100%` because `font: inherit` also sets font-size to inherit